e2e/flows/submit-hearing-requirements.flow.ts

```diff
@@ -98,7 +98,7 @@ export class SubmitHearingRequirementsFlow {
         await this.ccdFormPage.click('Continue');
 
         await this.ccdFormPage.setFieldValue(
-            'Does the appellant have any physical or mental health issues that may impact them during the hearing?',
+            'Does the appellant have any physical or mental health issues that may impact them',
```

Comment:
@Lee365 Any Reason we Modified the Content here.

Reply:
Reverted this change after rebase from master.